service/templates/projects.html

```diff
@@ -126,6 +126,12 @@ <h2> Settings for [[displayName]]</h2>
         </div>
       </div>
     </div>
+    <div id="version-control-enabled" class="form-group">
+      <div class="col-lg-6">
+        <input type="checkbox" id="version-control-enabled-checkbox" ng-model="settingsFormData.version_control_enabled" />
+        <label for="version-control-enabled-checkbox">Enable Config Version Control</label>
```

Comment:
Can we add some help text here to indicate what this is?

Reply:
Are we going to wait to deploy this (or hide it) until documentation is done (given the href?)